packages/core/src/config/__tests__/fixtures/resolve-config/plugin-esm.mjs

```diff
@@ -0,0 +1,10 @@
+export default async function () {
+  return {
+    id: 'test-plugin',
+    rules: {
+      oas3: {
+        'operation-3xx-response': 'warn',
```

Comment:
Technically this should be a function, not a severity level. Can we change it to something that won't mislead anyone?